include/libsemigroups/fpsemi-examples.hpp

```diff
@@ -448,6 +449,51 @@ namespace libsemigroups {
     //! \throws LibsemigroupsException if `r = 0`
     std::vector<relation_type> monogenic_semigroup(size_t m, size_t r);
 
+    //! A presentation for the cyclic inverse monoid.
+    //!
+    //! Returns a vector of relations giving a monoid presentation defining
+    //! the cyclic inverse monoid of degree `n`.
+    //!
+    //! The combination of `val` and `index` determines the specific
+    //! presentation which is returned. The options for these are:
+    //! * `val = author::Fernandes`
+    //!    * `index = 0` (see Theorem 2.6 of [10.48550/arxiv.2211.02155][])
+    //!    * `index = 1` (see Theorem 2.7 of [10.48550/arxiv.2211.02155][])
+    //!
```

Comment:
Still seems to be missing the number of generators and relations, same for the other bit of doc, otherwise, looks good!